content/components/side-nav/default.md

```diff
@@ -79,15 +79,17 @@ code:
       />
 ---
 
-The side navigation is used to provide a means for users to find other pages which share a similar topic or section.
+The side navigation allows users to find other pages which share a similar topic or section. It supports three levels of nesting along with an accompanying heading.
```

Comment:
Maybe we need to say "by default it supports three levels of nesting" so people know it can be tweaked.